website/justfile

```diff
@@ -0,0 +1,5 @@
+zola:
+	zola serve --open
+
+tailwind:
+	fnm exec --using latest npx @tailwindcss/cli -w -i input.css -o static/main.css
```

Comment:
nit: could this be changed to respect a `.node-version` or the `engines` field of `package.json`, or something like that?

Reply:
just not sure if the recommended workflow should be having to use `fnm`, I feel like using normal `npx` and then guaranteeing a consistent version via some other means is better

Reply:
`fnm` is just what I personally use. I removed it. It uses `npx` directly now.